site/sass/custom.scss

```diff
@@ -2,3 +2,8 @@
 .content pre > code {
   padding: unset;
 }
+
+// Fix the background color of the juice theme
+.content pre {
+    background-color: var(--code-background-color) !important;
```

Comment:
The `!important` is annoying, but Juice is applying a white background directly on the element so I can't override it otherwise...